iroha/src/peer.rs

```diff
@@ -66,6 +65,7 @@ struct PeerState {
     pub peers: HashSet<PeerId>,
     pub sent_pings: HashMap<Ping, Duration>,
     pub listen_address: String,
+    pub tx_queue: Arc<Mutex<Queue>>,
```

Comment:
Let's use fully qualified names?
You may add this requirement to `CONTRIBUTING.md`